packages/block-library/src/navigation/index.php

```diff
@@ -85,6 +85,37 @@ function block_core_navigation_build_css_font_sizes( $attributes ) {
 	return $font_sizes;
 }
 
+/**
+ * Renders a Navigation Block derived from data from the theme_location assigned
+ * via the block attribute 'location'.
+ *
+ * If the theme doesn't explicity support 'block-nav-menus' or no location was provided
+ * as a block attribute then an empty string is returned.
+ *
+ * @param  array $location The location of the classic menu to display.
+ * @param  array $attributes Navigation block attributes.
+ * @return string|false HTML markup of a generated Navigation Block or false if no location is specified.
+ */
+function render_classic_location_menu( $location, $attributes ) {
```

Comment:
Can we rename this to `render_menu_from_location` or something to that effect?

Reply:
Ah, it should probably be prefixed with `gutenberg_` too.

Reply:
Done here: https://github.com/WordPress/gutenberg/pull/32918